### DISCUSSION TITLE
DiscoDOS is the geekiest DJ tool on the planet

### DISCUSSION THREAD

**Discussion by JOJ0** [joalla/discogs_client#41] [Show & Tell]
DiscoDOS (https://github.com/JOJ0/discodos) is a geeky CLI tool for DJ's and Vinyl lovers but it could be so much more, if I would find the time to advance in new directions ;-) Ideas and contributions welcome!

If you want to see how it looks and feels, there is some youtube videos too: https://www.youtube.com/playlist?list=PLcHqk0rpp8bprmYlaXdrs6pbOpPoJwW-T

And it recently made it on libreav: https://libreav.org/software/discodos

**Comment by AnssiAhola** [comment]
That's a cool tool 👍 
Just a heads up, the Source and Announcements links at the end of libreav page, link to wrong user (J0J0) and endup in 404
Wrong user in "Developer(s)" as well

**Reply by JOJ0** [reply]
ah alright I didn't notice, will inform the maintainer of the libreav site!!!

**Reply by JOJ0** [reply]
thanks again for the hint. this has been fixed by the libreav site maintainer 😄

**Comment by JOJ0** [comment]
Because you were saying this in the projects thread:

> Now that I think about it, I might do some personal project with this, not sure what tho 🤔
> Any ideas? 😄

Project ideas? If you don't have a new project in mind already, I have a job for you. Hahaha. Have a look at DiscoDOS, maybe _you_ have an idea what it is missing or in what direction it could go? I was thinking, since you brought in the sales listing features, I could add a `disco sell` command easily that searches in users collection and lists it for sale. It's a little out of scope in a dj/playlist tool but actually sometimes I came across in the process of documenting my mixes that actually a record could just go and won't be put in a future mix anymore...

**Reply by JOJ0** [reply]
Finally released 1.0 last week and it got it's own website now: http://discodos.jojotodos.net
😀
